source/includes/substitute-placeholders.rst

```diff
@@ -0,0 +1,4 @@
+.. note::
+
+    You must substitute all values in quotes with angle brackets with
+    your KMS configuration values. 
```

Comment:
Suggestion: Maybe add an example replacement? Definitely don't have to use tabs this is just the first thing that came to mind. 

```suggestion
    your KMS configuration values. 
    
    For example, if your username is "MongoUser", you should replace the text in the following code snippet as follows:
    
    .. tab :: unreplaced
     
       .. code-block:: java
    
          "<your username>"

      .. tab :: replaced 
     
       .. code-block:: java
    
          "MongoUser"    
    
```